networkx/algorithms/centrality/tests/test_decay_centrality.py

```diff
@@ -0,0 +1,327 @@
+"""
+Unit tests for decay centrality
+
+"""
+
+import pytest
+
+import networkx as nx
+
+
+def test_decay_centrality_K():
+    """
+    Test the decay centrality of the krackhardt kite graph.
+
+    """
```

Comment:
Can you put the comments on all one line with triple " on that same line? (trying to reduce the number of blank lines)